src/category_theory/abelian/exact.lean

```diff
@@ -38,24 +38,42 @@ variables {X Y Z : C} (f : X ⟶ Y) (g : Y ⟶ Z)
 
 local attribute [instance] has_equalizers_of_has_kernels
 
+/--
+In an abelian category, a pair of morphisms `f : X ⟶ Y`, `g : Y ⟶ Z` is exact
+iff `image_subobject f = kernel_subobject g`.
+-/
+theorem exact_iff'' : exact f g ↔ image_subobject f = kernel_subobject g :=
```

Comment:
Can we add a `tfae` package for `exact_iff` and `exact_iff''`?

Reply:
done